docs/developers-guide/visual-tests.md

```diff
@@ -8,20 +8,27 @@ We use [Percy](https://percy.io/) via Github actions to run visual regression te
 
 ## How to run visual tests on CI
 
-Percy tests are supposed to be run on CI since every run is attached to a pull request. Only when a cypress tests run command is prefixed by `percy exec --` and there is a valid `PERCY_TOKEN` environment variable specified, Percy CLI will submit pages snapshots to Percy servers, and we will be charged for every screenshot. To make use of Percy more cost-efficient, we manually trigger visual tests by assigning `visual` pull request label. It will also run the tests on every subsequent commit. If you plan to perform a lot of commits while the PR is work-in-progress and you don't need to run tests on all of them, then just temporarily remove the label. It is important because we pay for every screenshot and it saves screenshot credits. Alternatively, you can trigger visual tests manually by posting a PR comment with a `@metabase-bot run visual tests` command.
-In addition to that, we need to ensure that underlying Cypress tests are valid, so we run them without submitting screenshots to Percy on every commit.
+Percy tests are intended to be run on CI since every run is attached to a pull request. Cypress tests will submit page snapshots to the Percy servers only when the Cypress run command is prefixed with `percy exec --` and a valid `PERCY_TOKEN` environment variable is set. Percy charges by the screenshot, so we do not do this by default.
+
+Instead, to reduce cost, we only trigger visual tests by manually adding the `visual` label to a pull request. If you plan to perform a lot of commits while the PR is a work-in-progress and you don't need to run tests on each commit, then just temporarily remove the label until you're ready since Percy will run every time you push a new commit to a PR with a `visual` label. Preventing unnecessary Percy runs saves money.
+
+Alternatively, you can trigger visual tests manually by posting a PR comment with a `@metabase-bot run visual tests` command.
+
+If there is no `visual` label, the tests will run to ensure that the underlying Cypress tests are valid, but we will not submit screenshots to the Percy servers.
```

Comment:
```suggestion
When your PR is ready for review, you can run visual tests (handled by Percy) in one of two ways:

- Add a `visual` label to the pull request.
- Comment on the pull request with `@metabase-bot run visual tests`

**Only run these visual tests when your pull request is ready for review**. Percy charges by the screenshot, so running the visual tests on every commit gets expensive.

By default, Cypress tests will run to ensure that the underlying tests are valid, but Cypress won't submit screenshots to the Percy servers. When the PR is labeled or commented to run the visual tests, the Cypress run command is prefixed by `percy exec --` and a valid `PERCY_TOKEN` environment variable, and Cypress will submit the screenshots to the Percy servers.
```
Took a pass at this to see if I understand how this works. This edit clarifies how to get the visual tests to run,  emphasizes when to run viz tests to save money, and explains how adding a label changes the default behavior.